app/mailers/school_mailer.rb

```diff
@@ -326,4 +327,14 @@ def nqt_plus_one_sit_invite(recipient:, start_url:, school:)
       },
     ).tag(:year2020_invite).associate_with(school, as: :school)
   end
+
+  def diy_wordpress_notification(user:)
+    template_mail(
+      DIY_WORDPRESS_NOTIFICATION_TEMPLATE,
+      to: user.email,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {},
+    ).tag(:diy_wordpress_notification).associate_with(user, as: :user)
+  end
```

Comment:
Why does it have `wordpress` in the name - it makes me want to just delete it :D

Reply:
The good news is this email is telling them we're getting rid of the wordpress site

Reply:
:partying_face: